public_website/views.py

```diff
@@ -165,11 +165,11 @@ def get(self, request):
             questions_queryset = []
 
             if 'answered' in question_categories:
-                answered_questions = result.filter(answers__approved_by__isnull=False)
+                answered_questions = result.filter(answers__status='published')
                 questions_queryset = answered_questions
 
             if 'unanswered' in question_categories:
-                unanswered_questions = result.filter(answers__approved_by__isnull=True)
+                unanswered_questions = result.exclude(answers__status='published')
```

Comment:
Will this change pick questions with 'draft' answers?

Reply:
Hello @badrihippo can you check this?

Reply:
Okay, so this will pick questions with 'draft' and 'under review' answers, as well as questions with no answers at all. Does that behaviour need to be changed?

Reply:
No, it seems correct! My bad with this one, this is good to go!